src/translations/en.json

```diff
@@ -253,6 +253,7 @@
     "createNetworkModificationNode": "Create a network modification node",
     "createModelNode": "Create a model node",
     "removeNode": "Remove node",
+    "exportNode": "Export node",
```

Comment:
It is not the node that is exported but the associated network

Reply:
New proposal

Reply:
Export network ?

Reply:
Export Case PO approval